worker/dockerworker/dockerworker.go

```diff
@@ -53,6 +53,16 @@ func (d *dockerworker) ConfigureRun(state *run.State) error {
 		}
 	}
 
+	workerLocationJson, err := json.Marshal(state.WorkerLocation)
+	if err != nil {
+		return fmt.Errorf("Error encoding worker location: %v", err)
+	}
+
+	state.WorkerConfig, err = state.WorkerConfig.Set("workerLocation", string(workerLocationJson))
+	if err != nil {
+		return fmt.Errorf("Could not set worker location in the worker config: %v", err)
+	}
+
```

Comment:
:100:

Passing the value as already-encoded JSON is great, as it discourages the worker implementer from trying to add properties :)